static/app/components/profiling/FlamegraphZoomView.tsx

```diff
@@ -121,72 +119,54 @@ function FlamegraphZoomView({
     setGridRenderer(newGridRenderer);
 
     function clearOverlayCanvas() {
-      // We are doing this because of typescript, in reality we are creating a scope
-      // where flamegraphRenderer is not null (see check on L86)
-      if (flamegraphRenderer === null) {
-        return;
+      if (flamegraphRenderer) {
```

Comment:
Actually, I think my comment was wrong and why I removed it. Imagine if we at some point clear the flamegraph renderer but forget to clear the handlers, then our scheduler may still call the draw call and it would throw because flamegraphRenderer is now null